runtime/doc/vvars.txt

```diff
@@ -550,8 +550,7 @@ v:searchforward
 v:servername
 		Primary listen-address of Nvim, the first item returned by
 		|serverlist()|. Usually this is the named pipe created by Nvim
-		at |startup| or given by |--listen| (or the deprecated
-		|$NVIM_LISTEN_ADDRESS| env var).
+		at |startup| or given by |--listen|.
```

Comment:
dont think this should be changed yet. not sure about the final plan for NVIM_LISTEN_ADDRESS, it might be un-deprecated for this particular purpose (which is different than its previous dual/conflicting role)